ros2topic/ros2topic/verb/list.py

```diff
@@ -22,7 +22,6 @@ class ListVerb(VerbExtension):
     """Output a list of available topics."""
 
     def add_arguments(self, parser, cli_name):
-        add_arguments(parser)
```

Comment:
This change doesn't seem to be related to the topic of the PR. Please consider separating it.

Reply:
See 5a28d9da97d673a795a3e911cd6e843765399065.